aaad/marathon.py

```diff
@@ -17,13 +17,13 @@ def getName(self):
         return "Marathon"
 
     def filterResponseBody(self, body, allowed_namespaces, request_uri):
-        uri_pattern = re.compile("^{}$".format("/marathon/*v2/apps"))
+        uri_pattern = re.compile("^{}$".format("/marathon/*v2/apps.*"))
```

Comment:
So `/marathon/*v2/apps.*` implies that the following are accepted:

- /marathon/v2/apps
- /marathonv2/apps
- /marathon/////v2/apps

Is that the intent?